libc/src/math/generic/sin.cpp

```diff
@@ -231,15 +147,21 @@ LLVM_LIBC_FUNCTION(double, sin, (double x)) {
   DoubleDouble sin_k_cos_y = fputil::quick_mult<NO_FMA>(cos_y, sin_k);
   DoubleDouble cos_k_sin_y = fputil::quick_mult<NO_FMA>(sin_y, cos_k);
 
-  FPBits sk_cy(sin_k_cos_y.hi);
-  FPBits ck_sy(cos_k_sin_y.hi);
```

Comment:
were these just unused before? Do we not have `-Wunused` turned on? We ought to.

Reply:
I don't know why there were no warnings for these unused variables.

Reply:
https://github.com/llvm/llvm-project/issues/96689